bundler/exe/bundle

```diff
@@ -18,14 +18,17 @@ end
 # Workaround for non-activated bundler spec due to missing https://github.com/rubygems/rubygems/commit/4e306d7bcdee924b8d80ca9db6125aa59ee4e5a3
 gem "bundler", Bundler::VERSION if Gem.rubygems_version < Gem::Version.new("2.6.2")
 
-# Check if an older version of bundler is installed
-$LOAD_PATH.each do |path|
-  next unless path =~ %r{/bundler-0\.(\d+)} && $1.to_i < 9
-  err = String.new
-  err << "Looks like you have a version of bundler that's older than 0.9.\n"
-  err << "Please remove your old versions.\n"
-  err << "An easy way to do this is by running `gem cleanup bundler`."
-  abort(err)
+if Gem::VERSION < "3.2.3" && RUBY_VERSION < "2.6.a" && !ENV["BUNDLER_NO_OLD_RUBYGEMS_WARNING"]
```

Comment:
I think we need to use Gem::Versions here. For example, `"3.10.0" < "3.2.3" => true`.

Reply:
Nice catch, I'll update!

Reply:
Addressed!